src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/LspServices.cs

```diff
@@ -14,16 +14,12 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer;
 internal class LspServices : ILspServices
 {
     private readonly IServiceProvider _serviceProvider;
-    private readonly IEnumerable<IRazorStartupService> _startupServices;
     public bool IsDisposed = false;
 
     public LspServices(IServiceCollection serviceCollection)
     {
         serviceCollection.AddSingleton<ILspServices>(this);
         _serviceProvider = serviceCollection.BuildServiceProvider();
-
-        // Create all startup services
-        _startupServices = _serviceProvider.GetServices<IRazorStartupService>();
```

Comment:
This will break lots of things. This should be `_ = _serviceProvider.GetServices<IRazorStartupService>();` at the very least.